pygmt/tests/test_clib.py

```diff
@@ -808,16 +808,17 @@ def test_dataarray_to_matrix_dims_fails():
         dataarray_to_matrix(grid)
 
 
-def test_dataarray_to_matrix_inc_fails():
+def test_dataarray_to_matrix_irregular_inc_warning():
     """
-    Check that it fails for variable increments.
+    Check that it warns for variable increments.
```

Comment:
Maybe link to the issue at #1468 so people know why this is being tested like this.
```suggestion
    Check that it warns for variable increments, see also
    https://github.com/GenericMappingTools/pygmt/issues/1468.
```